### DISCUSSION TITLE
Discard selected files

### DISCUSSION THREAD

**Discussion by siulkilulki** [NeogitOrg/neogit#296] [Q&A]
Is it possible to discard selected files?

**Comment by azzamsa** [comment]
Came here for the same thing.

If I select multiple files like this, pressing `x` won't do anything

![image](https://github.com/NeogitOrg/neogit/assets/17734314/3e1f2872-ef7a-4a1a-86cc-28e840d8cc44)

Is this expected @CKolkey?

```
  "neogit": { "branch": "master", "commit": "66b29453df7050543a51986620f8961a79a13fcc" },
```

Related:
- https://github.com/NeogitOrg/neogit/issues/684
